site/components/BlockContentRenderer/index.tsx

```diff
@@ -101,6 +101,13 @@ const serializers: BlockContentProps["serializers"] = {
         </div>
       );
     },
+    pdf: (params: { node: { name: string; url: string } }) => {
+      return (
+        <Text t="body2" className={styles.paragraph}>
+          Download PDF <A href={`${params.node.url}?dl=`}>{params.node.name}</A>
```

Comment:
What do you think about allowing content writers to control the "Download PDF" CTA copy?
Could we see a use case where we might not want have this hardcoded?